src/test/scala/com/fluidcode/processing/silver/GetPostsInfoSpec.scala

```diff
@@ -4,29 +4,34 @@ import com.fluidcode.configuration.Configuration
 import org.apache.spark.sql.QueryTest
 import org.apache.spark.sql.delta.test.DeltaExtendedSparkSession
 import org.apache.spark.sql.test.SharedSparkSession
+import com.fluidcode.models.silver.PostsData
+import com.fluidcode.processing.bronze.BronzeLayer
+import com.fluidcode.processing.silver.GetPostsInfoAndCreateTable._
 
-case class SilverData(comments_disabled: Boolean, edge_media_preview_like_count: Long, edge_media_to_caption_edges_node_text: String,
-                         edge_media_to_comment_count: Long, gating_info: String, id: String, is_video: Boolean, location: String,
-                         owner_id: String, shortcode: String, tags: String, taken_at_timestamp: Long, username: String)
 
 class GetPostsInfoSpec extends QueryTest
   with SharedSparkSession
   with DeltaExtendedSparkSession  {
 
-  test("getPostsInfo should create  PostsInfoTable from Bronze Layer" ) {
+  test("getPostsInfo should select post info data from Bronze Layer" ) {
```

Comment:
this should be done in the other test class, not here. 
for each object or class, we need to have a mirroring test class, not 2.